UserAdminBundle/Form/Type/UserType.php

```diff
@@ -58,9 +57,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
-     * @param OptionsResolverInterface $resolver
+     * @param OptionsResolver $resolver
      */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
Could you also complete the test `UserTypeTest` ?

I checked it and I think that there are some checks missing